package.json

```diff
@@ -21,7 +21,7 @@
     "create-react-component": "yo react-domain"
   },
   "dependencies": {
-    "@theforeman/vendor": "^6.0.0",
+    "@theforeman/vendor": "^8.2.0",
```

Comment:
Is there any reason to use the old `@theforeman/vendor` version? It took me a while to figure out why some PF4 features don't work

Reply:
I think the 7.x branch was somehow broken, but 8.x should be fine.

@laviro @sharvit @MariaAga might know better

Reply:
Vendor 8.X includes some breaking changes to diff modal so if you need to update vendor please include the fixes for the diff.
There is a PR to upgrade to 8 including the fixes: https://github.com/theforeman/foreman/pull/7963